cypress/e2e/awx/views/schedules.cy.ts

```diff
@@ -186,13 +186,135 @@ describe('Schedules - Delete', () => {
 
 describe('Schedules - Edit', () => {
   //Make sure to assert the deletion by intercepting the Patch request
+  let schedule: Schedule;
+
+  before(() => {
+    cy.awxLogin();
+  });
+
+  beforeEach(() => {
+    cy.createAWXSchedule().then((sched: Schedule) => (schedule = sched));
+    cy.navigateTo('awx', 'schedules');
+  });
 
-  it.skip('can edit a simple schedule from details page', () => {});
-  it.skip('can edit a simple schedule from the schedules list row', () => {});
-  it.skip('can edit a schedule to add rules', () => {});
-  it.skip('can edit a schedule to exceptions', () => {});
-  it.skip('can edit a schedule to remove rules', () => {});
+  it('can edit a simple schedule from details page', () => {
+    cy.getTableRow('name', schedule.name).within(() => {
+      cy.get('[data-cy="name-column-cell"]').click();
+    });
+    cy.getBy('[data-cy="edit-schedule"]').click();
+    cy.get('[data-cy="wizard-nav"]').within(() => {
+      ['Details', 'Rules', 'Exceptions', 'Review'].forEach((text, index) => {
+        cy.get('li')
+          .eq(index)
+          .should((el) => expect(el.text().trim()).to.equal(text));
+      });
+    });
+    cy.get('[data-cy="description"]').type('-edited');
+    cy.clickButton(/^Next$/);
+    cy.clickButton(/^Next$/);
+    cy.clickButton(/^Next$/);
+    cy.get('[data-cy="description"]').contains('-edited');
+  });
+  it('can edit a simple schedule from the schedules list row', () => {
+    cy.getTableRow('name', schedule.name).within(() => {
+      cy.get('[data-cy="edit-schedule"]').click();
+    });
+    cy.get('[data-cy="wizard-nav"]').within(() => {
+      ['Details', 'Rules', 'Exceptions', 'Review'].forEach((text, index) => {
+        cy.get('li')
+          .eq(index)
+          .should((el) => expect(el.text().trim()).to.equal(text));
+      });
+    });
+    cy.get('[data-cy="description"]').type('-edited');
+    cy.clickButton(/^Next$/);
+    cy.clickButton(/^Next$/);
+    cy.clickButton(/^Next$/);
+    cy.get('[data-cy="description"]').contains('-edited');
+  });
+  it('can edit a schedule to add rules', () => {
+    cy.getTableRow('name', schedule.name).within(() => {
+      cy.get('[data-cy="edit-schedule"]').click();
+    });
+    cy.get('[data-cy="wizard-nav"]').within(() => {
+      ['Details', 'Rules', 'Exceptions', 'Review'].forEach((text, index) => {
+        cy.get('li')
+          .eq(index)
+          .should((el) => expect(el.text().trim()).to.equal(text));
+      });
+    });
+    cy.clickButton(/^Next$/);
+    cy.clickButton(/^Add rule$/);
+    cy.clickButton(/^Add$/);
+    cy.clickButton(/^Next$/);
+    cy.clickButton(/^Next$/);
+    cy.get('.pf-v5-c-page__main-body').contains('Rule 2');
```

Comment:
is there a `data-cy` available to use instead of a PF locator?

Reply:
There isn't a `data-cy` for this view.